tests/Agent/IntegrationTests/Shared/WCF/WcfService.cs

```diff
@@ -62,6 +63,18 @@ public async Task<string> TAPGetData(int value)
             return await DoWork(value.ToString(), false, false);
         }
 
+        public async Task<string> TAPMakeExternalCalls()
+        {
+            using (var httpClient = new HttpClient())
+            {
+                _ = await httpClient.GetAsync("https://google.com");
+                _ = await httpClient.GetAsync("https://bing.com");
+                _ = await httpClient.GetAsync("https://yahoo.com");
```

Comment:
Do we care about success or failure of these requests? or is that unimportant?

Reply:
It's unimportant - we just need to validate that the instrumentation of the calls happened, even if the requests failed it's not significant for this test.